ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/queue_monitor_length.md

```diff
@@ -32,3 +35,24 @@ interface Management1
    vrf MGMT
    ip address 10.73.255.122/24
 ```
+
+## Queue Monitor
+
+### Queue Monitor Length
+
+
+| Enabled | Logging Interval | Default Thresholds High | Default Thresholds Low | Notifying | tx-latency | CPU Thresholds High | CPU Thresholds Low |
+| ------- | ---------------- | :-----------------------: | :----------------------: | --------- | ---------- | :-------------------: | :------------------: |
+| True | 100 | 100 | 10 | True | True | - | - |
```

Comment:
Why are the two last columns empty when the config shows cpu thresholds?

Reply:
Issue in the template - should be resolved now.